stock_inventory_adjustment/models/stock_inventory.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class InventoryAdjustmentsGroup(models.Model):
+    _name = "stock.inventory"
+    _description = "Inventory Adjustment Group"
+    _order = "state, id desc"
+
+    name = fields.Char(required=True)
+
+    date = fields.Datetime(default=datetime.date.today())
+
+    state = fields.Selection(
+        [("2", "Draft"), ("1", "In Progress"), ("3", "Done")], default="2"
+    )
+
+    location_ids = fields.Many2many(
+        "stock.location", string="Location", domain="[('usage', '=', 'internal')]"
+    )
+
+    product_selection = fields.Selection(
+        [("all", "All Products"), ("manual", "Manual Selection")],
+        default="all",
+        required=True,
+    )
+
+    stock_quant_ids = fields.Many2many("stock.quant", string="Inventory Adjustment")
+
+    stock_move_ids = fields.One2many(
+        "stock.move", "inventory_adjustment_id", string="Inventory Adjustments Done"
+    )
+
+    count_stock_quants = fields.Integer(
+        compute="_compute_count_stock_quants", string="Adjustments"
+    )
+
+    count_stock_moves = fields.Integer(
+        compute="_compute_count_stock_moves", string="Stock Moves"
+    )
+
+    @api.depends("stock_quant_ids")
+    def _compute_count_stock_quants(self):
+        self.count_stock_quants = len(self.stock_quant_ids)
+
+    @api.depends("stock_move_ids")
+    def _compute_count_stock_moves(self):
+        sm_ids = self.mapped("stock_move_ids").ids
+        self.count_stock_moves = len(sm_ids)
+
+    def action_state_to_in_progress(self):
+        active_rec = self.env["stock.inventory"].search([("state", "=", "1")])
```

Comment:
I don't like the use of integer as state value as looking at this code, you cannot determine what state is behind

Reply:
Also done, thank you again!